<ui version="4.0" >
 <class>HelloFileBrowser</class>
 <widget class="QWidget" name="HelloFileBrowser" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>402</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QSplitter" name="splitter_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="TQtObjectViewFrame" name="treeView" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Select the histogram object </string>
        </property>
        <property name="whatsThis" >
         <string>Tree view of the TDirectory object</string>
        </property>
       </widget>
       <widget class="QWidget" native="1" name="widget_3" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
          <horstretch>7</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" >
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QSplitter" name="splitter" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="TQtWidget" native="1" name="widget" />
           <widget class="TQtWidget" native="1" name="widget_2" />
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox" >
       <item>
        <property name="text" >
         <string>.x hsimple.C</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>.q</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TQtWidget</class>
   <extends>QWidget</extends>
   <header location="global" >TQtWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TQtObjectViewFrame</class>
   <extends>QTreeView</extends>
   <header>TQtObjectViewFrame.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
